components/prism-factor.js

```diff
@@ -0,0 +1,277 @@
+(function (prism) {
```

Comment:
that would have shadowed the `Prism` with which the IIFE is called, and I avoid shadowing whenever possible. I'll change it since you point it out though.